package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "loopback.io-workflow-scripts",
+  "version": "1.0.0",
+  "description": "This file: Node-dependant workflow scripts for the jekyll-based site",
+  "scripts": {
+    "fetch-readmes": "get-readmes --repos=./_data --out=./_includes/readmes"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Strongloop/loopback.io.git"
+  },
+  "devDependencies": {
+    "getreadmes": "github:sequoia/get-readmes"
```

Comment:
I merged the PR in get-readmes, so would you please go ahead and make the change here?